app/views/deliveries/index.html.erb

```diff
@@ -0,0 +1,107 @@
+<head>
+  <style>
+  
+  .waiting_on{
+    background-color:#fafad2;
+  }
+
+  .received{
+    background-color:#90ee8f;
+  }
+  </style>
+</head>
+
+<body>
+
+<div>
+  <div>
+    <h1>
+      Delivery Tracker
+    </h1>
+
+    <p>Know if something gets lost in the mail.</p>
+  </div>
+</div>
+
+<hr>
+
+<div>
+  <div>
+    <h2>
+      Expecting a package?
+    </h2>
+
+    <form action="/insert_delivery" method="post">
+    <input type="hidden" name="query_user_id" value="<%=current_user.id%>">
+      <div>
+        <label for="description_box">
+          Description
+        </label>
+
+        <input type="text" id="description_box" name="query_description">
+      </div>
+
+      <div>
+        <label for="supposed_to_arrive_on_box">
+          Supposed to arrive on
+        </label>
+
+        <input type="date" id="supposed_to_arrive_on_box" name="query_supposed_to_arrive_on">
+      </div>
+
+      <div>
+        <label for="details_box">
+          Details
+        </label>
+
+        <textarea type="text" id="details_box" name="query_details" rows="3"></textarea>
+      </div>
+
+      <button>
+        Log delivery
+      </button>
+    </form>
+  </div>
+</div>
+
+<hr>
+
+<div class="waiting_on">
+  <h2>Waiting on</h2>
+
+<ul>
+<% @list_of_deliveries.where(:user_id => current_user.id).where(:arrived => false).each do |a_delivery| %>
```

Comment:
This is one option of getting the current deliveries that have not arrived yet, but to not have to do this query in the html, you can also do it in the controller and pass the array in as a variable!